<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealTime</class>
 <widget class="QDialog" name="RealTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>635</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>635</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1020</width>
    <height>635</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Real Time Controller</string>
  </property>
  <widget class="QToolBox" name="main_toolBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>1021</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>577</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Analog/Digital Output</string>
    </attribute>
    <widget class="QGroupBox" name="Dout_group">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>0</y>
       <width>381</width>
       <height>571</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>571</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>481</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Digital Channels</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Aout_group">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>381</width>
       <height>571</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>571</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>481</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Analog Channels</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Update_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Update ADwin</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Profile_group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>191</width>
       <height>261</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QListWidget" name="Profile_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>192</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Test1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Test2</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="Profile_del_button">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Profile_load_button">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Profile_new_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>577</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Analog Input</string>
    </attribute>
    <widget class="QTextEdit" name="State">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>450</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Channel_Dial">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>450</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Vertical Adjustment</string>
     </property>
     <widget class="QDial" name="V_dial">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>81</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="notchTarget">
       <double>1.000000000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDial" name="Position_dial">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>51</width>
        <height>71</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>50</number>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="notchTarget">
       <double>10.000000000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="Position_dis">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="V_dis">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>85</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
      <property name="intValue" stdset="0">
       <number>5000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>90</y>
        <width>10</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </widget>
    <widget class="QDial" name="H_dial">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="notchTarget">
      <double>1.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="H_display">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>540</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>540</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="Displaywindow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1001</width>
       <height>441</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="Screen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>981</width>
        <height>421</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Screen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Cursor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>981</width>
        <height>421</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>BlankCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Cursor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="Back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>981</width>
        <height>421</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Backgroud</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <zorder>Back</zorder>
     <zorder>Screen</zorder>
     <zorder>Cursor</zorder>
    </widget>
    <widget class="QTabWidget" name="Channel_control">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>450</y>
       <width>311</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
